site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -1153,7 +1230,9 @@
         "id": "PTh1rtDn8r4-"
       },
       "source": [
-        "Here you can see that with class weights the accuracy and precision are lower because there are more false positives, but conversely the recall and AUC are higher because the model also found more true positives. Despite having lower accuracy, this model has higher recall (and identifies more fraudulent transactions). Of course, there is a cost to both types of error (you wouldn't want to bug users by flagging too many legitimate transactions as fraudulent, either). Carefully consider the trade-offs between these different types of errors for your application."
+        "Here you can see that with class weights the accuracy and precision are lower because there are more false positives, but conversely the recall and AUC are higher because the model also found more true positives. Despite having lower accuracy, this model has higher recall (and identifies more fraudulent transactions than the baseline model at threshold 50%). Of course, there is a cost to both types of error (you wouldn't want to bug users by flagging too many legitimate transactions as fraudulent, either). Carefully consider the trade-offs between these different types of errors for your application.\n",
```

Comment:
It might be worth refering back to the fact, from the  previous comment, that this is equivalent to changing the threshold. And the different metric values here are mostly because of that skewing. 

To emphasize this it could be worth while to put markers on the curve plots at 10, 50, 90% thresholds so you can see that some of it is just moving the threshold values around.